diff -ur libcdio-0.79/include/cdio++/iso9660.hpp libcdio-0.79.new/include/cdio++/iso9660.hpp
--- libcdio-0.79/include/cdio++/iso9660.hpp     2006-03-07 22:04:01.000000000 +0100
+++ libcdio-0.79.new/include/cdio++/iso9660.hpp 2008-02-14 14:53:36.000000000 +0100
@@ -31,6 +31,7 @@
 #include <cdio++/cdio.hpp>
 #include <string.h>
 #include <vector>		// vector class library
+#include <cstdlib>
 using namespace std;
 
 /** ISO 9660 class.
