diff -up libcdio-0.80/Makefile.am.bak libcdio-0.80/Makefile.am
--- libcdio-0.80/Makefile.am.bak	2008-03-15 17:21:28.000000000 +0100
+++ libcdio-0.80/Makefile.am	2008-06-04 17:23:13.942500222 +0200
@@ -20,6 +20,10 @@
 ##   which configure then turns into a Makefile  ...
 ##     which make can then use to produce stuff. Isn't configuration simple? 
 
+if BUILD_CD_PARANOIA
+paranoiadist = libcdio_paranoia.pc.in 
+endif
+
 EXTRA_DIST = \
 	MSVC/README MSVC/cd-info.vcproj \
 	MSVC/config.h \
@@ -34,7 +38,8 @@ EXTRA_DIST = \
 	libiso9660.pc.in \
 	libiso9660++.pc.in \
 	libudf.pc.in \
-	package/libcdio.spec.in 
+	package/libcdio.spec.in \
+	$(paranoiadist)
 
 SUBDIRS = doc include lib src test example
 
@@ -48,7 +53,7 @@ pkgconfig_DATA = libcdio.pc  \
 	         libcdio++.pc  \
 	         libiso9660.pc \
 	         libiso9660++.pc \
-	         libudf.pc
+	         libudf.pc \
 		 $(paranoiapcs)
 
 
