diff -Naru libcdio-0.70.orig/lib/cdio.c libcdio-0.70.new/lib/cdio.c
--- libcdio-0.70.orig/lib/cdio.c	2004-08-29 06:55:24.000000000 +0300
+++ libcdio-0.70.new/lib/cdio.c	2005-03-18 15:48:40.000000000 +0200
@@ -848,7 +848,7 @@
   if (cdio == NULL) return -1;
   
   if (cdio->op.read)
-    return cdio->op.read (cdio->env, buf, size);
+    return (cdio->op.read) (cdio->env, buf, size);
   return -1;
 }
 
diff -Naru libcdio-0.70.orig/lib/_cdio_stream.c libcdio-0.70.new/lib/_cdio_stream.c
--- libcdio-0.70.orig/lib/_cdio_stream.c	2004-02-07 20:53:02.000000000 +0200
+++ libcdio-0.70.new/lib/_cdio_stream.c	2005-03-18 15:46:44.000000000 +0200
@@ -145,7 +145,7 @@
 
   if (!_cdio_stream_open_if_necessary(obj)) return 0;
 
-  read_bytes = obj->op.read(obj->user_data, ptr, size*nmemb);
+  read_bytes = (obj->op.read)(obj->user_data, ptr, size*nmemb);
   obj->position += read_bytes;
 
   return read_bytes;
diff -Naru libcdio-0.70.orig/src/cd-drive.c libcdio-0.70.new/src/cd-drive.c
--- libcdio-0.70.orig/src/cd-drive.c	2004-08-27 14:53:38.000000000 +0300
+++ libcdio-0.70.new/src/cd-drive.c	2005-03-18 15:51:43.000000000 +0200
@@ -140,7 +140,6 @@
 
 /* CDIO logging routines */
 
-static cdio_log_handler_t gl_default_cdio_log_handler = NULL;
 
 static void 
 _log_handler (cdio_log_level_t level, const char message[])
diff -Naru libcdio-0.70.orig/src/cd-info.c libcdio-0.70.new/src/cd-info.c
--- libcdio-0.70.orig/src/cd-info.c	2004-09-02 04:33:57.000000000 +0300
+++ libcdio-0.70.new/src/cd-info.c	2005-03-18 15:53:49.000000000 +0200
@@ -371,7 +371,6 @@
 
 /* CDIO logging routines */
 
-static cdio_log_handler_t gl_default_cdio_log_handler = NULL;
 #ifdef HAVE_CDDB
 static cddb_log_handler_t gl_default_cddb_log_handler = NULL;
 #endif
diff -Naru libcdio-0.70.orig/src/iso-info.c libcdio-0.70.new/src/iso-info.c
--- libcdio-0.70.orig/src/iso-info.c	2004-06-19 22:15:15.000000000 +0300
+++ libcdio-0.70.new/src/iso-info.c	2005-03-18 15:55:57.000000000 +0200
@@ -133,7 +133,6 @@
 
 /* CDIO logging routines */
 
-static cdio_log_handler_t gl_default_cdio_log_handler = NULL;
 
 static void 
 _log_handler (cdio_log_level_t level, const char message[])
