--- a/lib/driver/cdtext.c	2012-03-23 15:30:24.002795726 +0100
+++ b/lib/driver/cdtext.c	2012-03-23 15:30:59.581851545 +0100
@@ -237,6 +237,7 @@
       i_track = p_data->i_track;
 
       for( j=0; j < CDIO_CDTEXT_MAX_TEXT_DATA; (p_data->bDBC ? j+=2 : j++) ) {
+	if (i_track > 99) break;
         if( p_data->text[j] == 0x00 && (!p_data->bDBC || p_data->text[j+1] == 0x00)) {
           
           /* omit empty strings */
